src/data-dictionary/events/K8sHpaSample/a_template.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
seems like we've got some template ones in here

Reply:
Yes, this is now removed.